README.md

```diff
@@ -178,7 +227,31 @@ run:
     python3 -m unittest tests.test_stuff.TestFactParsing
     python3 -m unittest tests.test_stuff.TestFactParsing.test_plain_name
 
-#### Migrating from hamster-applet
+##### Flatpak
+
+To run the tests inside the flatpak, use:
+
+```bash
+flatpak-builder --run build/flatpak org.gnome.Hamster.json \
+    python3 -m unittest
+```
+
+#### Migrating
+
+##### Migrating data to flatpak
+
+If you would like to retain your data from a non-flatpak installation,
+you can do so by running:
+
+```bash
+gio copy -b \
+    ~/.local/share/hamster/hamster.db \
+    ~/.var/app/org.gnome.Hamster/data/hamster/
```

Comment:
Why use `gio copy` here? Looks like these are two regular paths that can just use normal `cp`?